WordPressKitTests/ReaderSiteServiceRemoteTests.swift

```diff
@@ -289,12 +289,12 @@ class ReaderSiteServiceRemoteTests: XCTestCase {
         XCTAssertTrue(failure)
     }
 
-    func testCheckSiteExistsAtURLSuccess() {
+    func testCheckSiteExistsAtURLSuccess() throws {
         let testURLString = "http://www.wordpress.com"
         let testURL = URL(string: testURLString)!
+        let stubPath = try XCTUnwrap(OHPathForFile("empty.json", type(of: self)))
         stub(condition: {request in request.url?.absoluteString == testURLString}) { _ in
-            let stubPath = OHPathForFile("empty.json", type(of: self))
-            return fixture(filePath: stubPath!, headers: ["Content-Type" as NSObject: "application/json" as AnyObject])
+            return fixture(filePath: stubPath, headers: ["Content-Type" as NSObject: "application/json" as AnyObject])
```

Comment:
The header dictionary for content type JSON is repeated all over the place.

It could be DRYed in some form. I decided to keep it out because this PR is already noisy. Besides, the focus of this stream of work is to prepare for SPM. DRYing this definition would be useful, but doesn't bring us closer to that goal.